### PR TITLE
[Remove Immediate] LPS-188559 Remove immediate property from DLProcessorRegistryImpl

### PR DESCRIPTION
**Pull request by MrJohn1911**
Previous pull: https://github.com/liferay-core-infra/liferay-portal/pull/6060

Ticket: Ticket: https://liferay.atlassian.net/browse/LPS-188559

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by MrJohn1911** [comment]
ci:test:document

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
ci:test:document

**Comment by Kvale1733** [comment]
ci:test:relevant

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/view_file_entry_preview_error.jsp [hunk 1]
```diff
@@ -28,11 +28,11 @@ FileVersion fileVersion = (FileVersion)request.getAttribute(WebKeys.DOCUMENT_LIB
 				long maxFileSize = dlPreviewSizeException.getMaxFileSize();
 
 				if (maxFileSize == 0) {
-					maxFileSize = DLProcessorRegistryUtil.getPreviewableProcessorMaxSize(fileVersion.getGroupId());
+					maxFileSize = DLProcessorHelperUtil.getPreviewableProcessorMaxSize(fileVersion.getGroupId());
 				}
 				%>
 
-				<liferay-ui:message arguments="<%= LanguageUtil.formatStorageSize(maxFileSize, locale) %>" key="this-file-is-too-large-to-preview.-the-maximum-file-size-for-previews-is-x" />
+				<liferay-ui:message arguments="<%= LanguageUtil.formatStorageSize(maxFileSize, locale) %>" key="this-file-is-F-large-to-preview.-the-maximum-file-size-for-previews-is-x" />
```
Comment (rafaprax) [inline modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/view_file_entry_preview_error.jsp:35]: Hey @MrJohn1911 please fix that.

**Review by rafaprax — CHANGES_REQUESTED** [review]
Hey @MrJohn1911, please check the breaking change message format and fix if needed. Thx

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by Kvale1733** [comment]
@MrJohn1911 Regression found

Steps to reproduce: 
1. Go to System Settings > Documents & Media > File Preview Limits
2. Set Maxiumum file size to 1024
3. Go to Documents & Media
4. Upload a large file such as the one in our dependencies (super_large_file.pdf) located in 
portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/dependencies/
5. Click on the uploaded document to view it

Expected Results: The preview message should show something like
Correct

Actual Results: The preview message shown is

Regression

As you can see the message format is broken. I think maybe it's showing the language key instead?

**Comment by Kvale1733** [comment]
ci:close

**Comment by MrJohn1911** [comment]
ci:test:document

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
Hey @Kvale1733 I found this regression before my last push, so I fixed it, but I did not rerun the tests, so I will rerun now 😸

**Comment by Kvale1733** [comment]
ci:test:sf

**Comment by Kvale1733** [comment]
@MrJohn1911 No regressions found.

**Comment by dantewang** [comment]
Hi @rafaprax, regarding https://github.com/liferay-core-infra/liferay-portal/pull/6088/commits/56c6099bf3282b88e4052167974b45e0d8256444, I would suggest not renaming or renaming it to `DLProcessorManager`.

**Comment by dantewang** [comment]
@rafaprax Please ignore my last comment (I removed it). Is this PR ready? I still see it's in changes requested state, but it seems the changes are done.

**Review by rafaprax — APPROVED** [review]

**Comment by rafaprax** [comment]
Hey @dantewang, this pr is ready for review. Thx

**Review by dantewang — APPROVED** [review]

**Comment by dantewang** [comment]
ci:forward:force

cc @tinatian

**Pull request closed without merging**